doc/guides/maintaining-V8.md

```diff
@@ -146,54 +159,96 @@ process.
-* Once the fix has been merged upstream, it can be picked up during an update of the V8 branch, (see below).
+* Once the merge has been approved, it should be merged using the
+[merge script documented in the V8 wiki][V8MergingPatching]. Merging requires
+commit access to the V8 repository. If you don't have commit access you can
+indicate someone on the V8 team can do the merge for you.
+* It is possible that the merge request may not get approved, for example if it
+is considered to be a feature or otherwise too risky for V8 stable. In such
+cases we float the patch on the Node.js side. See the process on 'Backporting to
+Abandoned branches'.
+* Once the fix has been merged upstream, it can be picked up during an update of
+the V8 branch (see below).
 
 ### Backporting to Abandoned Branches
 
-Abandoned V8 branches are supported in the Node.js V8 repository. The fix needs
+Abandoned V8 branches are supported in the Node.js repository. The fix needs
 to be cherry-picked in the Node.js repository and V8-CI must test the change.
 
-* For each abandoned V8 branch corresponding to an LTS branch that is affected by the bug:
-    * Open a cherry-pick PR on nodejs/node targeting the appropriate *vY.x-staging* branch (e.g. *v6.x-staging* to fix an issue in V8-5.1).
-    * On Node.js < 9.0.0: Increase the patch level version in `v8-version.h`. This will not cause any problems with versioning because V8 will not publish other patches for this branch, so Node.js can effectively bump the patch version.
-    * On Node.js >= 9.0.0: Increase the `v8_embedder_string` number in `common.gypi`.
-    * In some cases the patch may require extra effort to merge in case V8 has changed substantially. For important issues we may be able to lean on the V8 team to get help with reimplementing the patch.
-    * Run the Node.js [V8-CI](https://ci.nodejs.org/job/node-test-commit-v8-linux/) in addition to the [Node.js CI](https://ci.nodejs.org/job/node-test-pull-request/).
+* For each abandoned V8 branch corresponding to an LTS branch that is affected
+by the bug:
+    * Checkout a branch off the appropriate *vY.x-staging* branch (e.g.
+    *v6.x-staging* to fix an issue in V8 5.1).
+    * Cherry-pick the commit(s) from the V8 repository.
+    * On Node.js < 9.0.0: Increase the patch level version in `v8-version.h`.
+    This will not cause any problems with versioning because V8 will not publish
+    other patches for this branch, so Node.js can effectively bump the patch
+    version.
+    * On Node.js >= 9.0.0: Increase the `v8_embedder_string` number in
+    `common.gypi`.
+    * In some cases the patch may require extra effort to merge in case V8 has
+    changed substantially. For important issues we may be able to lean on the V8
+    team to get help with reimplementing the patch.
+    * Open a cherry-pick PR on `nodejs/node` targeting the *vY.x-staging* branch
+    and notify the `@nodejs/v8` team.
+    * Run the Node.js [V8 CI] in addition to the [Node.js CI].
```

Comment:
Can you mention the `test-v8` target in the Makefile  (and possibly `make-v8.sh`)? It takes quite long to actually run locally but mentioning it here at least leaves a pointer about how to update the CI.

Reply:
I can. Do you have a suggestion for the wording?

Reply:
Maybe <code>Note: The CI uses the \`test-v8\` target in the \`Makefile\`, which uses \`tools/make-v8.sh\` to reconstruct a git tree in the \`deps/v8\` directory to run tests.</code>